automatic/keepass.portable/tools/chocolateyInstall.ps1

```diff
@@ -2,12 +2,27 @@
 
 $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
+$pp = Get-PackageParameters
+
+$config = "$toolsDir\package_install.config"
```

Comment:
You don't need to remember the install path between install / upgrades as choco has a feature to do that:

```
useRememberedArgumentsForUpgrades - Use Remembered Arguments For Upgrades - when running upgrades, use arguments for upgrade that were used for installation ('remembered'). This is helpful when running upgrade for all packages. Available in 0.10.4+. This is considered in preview for 0.10.4 and will be flipped to on by default in a future release.
```